src/app/notifications/notifications.component.html

```diff
@@ -0,0 +1,16 @@
+<div *ngFor="let notification of allNotifications$ | async">
+  <div class="container text-center notification-bar {{ notification.priority?.toLowerCase() }}">
+    <div *ngIf="notification.type == 'SITEWIDE'">
+      <div class="container-fluid text-center" *ngIf="!notificationDismissed(notification)" data-cy="notification-banner">
```

Comment:
To elaborate on Gary's comment, it's generally not good to have a function invocation in an ngIf, because it constantly gets evaluated.

Reply:
https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Onboarding#pitfalls-and-best-practices  number 3